android/app/src/main/res/layout/password.xml

```diff
@@ -1,27 +1,84 @@
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent"
-        app:layout_constraintVertical_bias="0.0"/>
-</androidx.constraintlayout.widget.ConstraintLayout>
\ No newline at end of file
+        android:layout_marginRight="24dp">
+
+        <TextView
+            android:id="@+id/tvTitle"
+            style="?android:windowTitleStyle"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginEnd="@dimen/fab_dialog_margin_right"
+            android:layout_marginRight="@dimen/fab_dialog_margin_right"
+            android:layout_marginTop="20dp"
+            android:maxLines="3"
+            android:singleLine="false"
+            android:text="@string/lorem"
+            app:layout_constraintEnd_toStartOf="@+id/btnRename"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent"/>
+
+        <com.google.android.material.floatingactionbutton.FloatingActionButton
+            android:id="@+id/btnRename"
+            style="@style/FAB.Dialog"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginBottom="@dimen/fab_dialog_margin_bottom"
+            android:layout_marginEnd="@dimen/fab_dialog_margin_right"
+            android:layout_marginRight="@dimen/fab_dialog_margin_right"
+            android:layout_marginTop="@dimen/fab_dialog_margin_top"
+            android:clickable="true"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:srcCompat="@drawable/ic_edit_24dp"/>
+
+    </androidx.constraintlayout.widget.ConstraintLayout>
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
+
+        <TextView
+            android:id="@+id/textEnterPassword"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginLeft="24dp"
+            android:layout_marginStart="24dp"
+            android:layout_marginTop="8dp"
+            android:text="@string/Enter_password"
+            app:layout_constraintHorizontal_chainStyle="spread"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent"/>
+
+        <EditText
+            android:id="@+id/etPassword"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginBottom="8dp"
+            android:layout_marginEnd="24dp"
+            android:layout_marginLeft="24dp"
+            android:layout_marginRight="24dp"
+            android:layout_marginStart="24dp"
+            android:layout_marginTop="8dp"
```

Comment:
The app style has no margin between labels and EditTexts, e.g. see new_wallet.xml.